From a61376f9afbd24a71af14137260e590a0c7e912e Mon Sep 17 00:00:00 2001
From: Murala S Swamy <ssmurala@habana.ai>
Date: Sun, 08 Dec 2024 14:29:32 +0200
Subject: [PATCH] [FSW-11035] fs1: align to specs 1p28

PCIe master interface path is updated in new
specs 1p28, it was taken care with this patch.

Change-Id: I98b56c94b1e55774b62dbe67c1b91a041c94d6b0
---

diff --git a/arch/fs1/arc/dev_security.c b/arch/fs1/arc/dev_security.c
index 92a4ac6..e5ebf46 100644
--- a/arch/fs1/arc/dev_security.c
+++ b/arch/fs1/arc/dev_security.c
@@ -85,8 +85,8 @@
 
 struct security_initiator_ctx initiator_data[SECURITY_INITIATOR_MAX] = {
 	[SECURITY_INITIATOR_PCIE] = {
-		mmIO_0_CFG_HALF_0_CXL_EP_PCIE_MSTR_IF_LBW_RR_BASE,
-		mmIO_0_CFG_HALF_0_CXL_EP_PCIE_MSTR_IF_LBW_AXI_OVRD_AXUSER_BASE,
+		mmIO_0_CFG_HALF_0_CXL_EP_PCIE_MSTR_IF_LBW_TSRCLD_R_BASE,
+		mmIO_0_CFG_HALF_0_CXL_EP_PCIE_MSTR_IF_LBW_TSRCLD_AXI_OVRD_AXUSER_BASE,
 	},
 	[SECURITY_INITIATOR_I2C_S_MSTR] = {
 		mmIO_0_CFG_HALF_0_PSOC_PSOC_CFG_I2C_S_MSTR_IF_LBW_RR_BASE,
